.github/workflows/ci-actions.yml

```diff
@@ -1,8 +1,29 @@
 name: Jakarta Contexts and Dependency Injection CI
 
 on:
+  push:
+    branches-ignore:
```

Comment:
Why keep below listed combinations in place?
That combination doesn't even match anything in this project, or am I missing something?

Reply:
The idea is to trigger the build on a pull request, to be sure  the build is going well, and no test is failed. And for some branches we can skip running the ci,  if we make a wip/ branche the build will not be triggered, this is a way is to prevent builds on all branches and to save cpu and memory for Github Actions